src/core/lib/layer.js

```diff
@@ -357,7 +370,11 @@ export default class Layer {
 
     // Initialize state only once
     this.setState({
-      attributeManager: new AttributeManager({id: this.props.id}),
+      attributeManager: new AttributeManager({
+        id: this.props.id,
+        gl: this.context.gl,
+        animation: this.props.animation
```

Comment:
What is this `animation` prop, that is only read once? Is this actually supposed to be `prop.transition`? 

Regardless I think we can avoid supplying it in the constructor per other comments.

Reply:
Yes it should be `transition`.